Fancyauth/Fancyauth.cs

```diff
@@ -46,12 +47,23 @@ public Fancyauth()
                 var rawServer = await FixIce.FromAsync(1, meta.begin_getServer, meta.end_getServer);
                 #endif
 
+                Console.Write("Steam user: ");
+                var steamUser = Console.ReadLine();
+                Console.Write("Steam password: ");
+                var steamPassword = Console.ReadLine();
```

Comment:
Are you really gonna type in username and password each time you start fancyauth? Not that you wouldn't forget to start it anyways ;)